app/api.py

```diff
@@ -124,7 +128,21 @@ def upload_file(self):
         service_request_id = request.args.get('service_request_id')
         file = request.files['file']
         filename = secure_filename(service_request_id + '_' + file.filename)
-        file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
+        fullpath = os.path.join(app.config['UPLOAD_FOLDER'], filename)
+        file.save(fullpath)
+
+        if S3_BUCKET:
+            bucket = S3_BUCKET
+            s3_client = boto3.client('s3')
+            try:
+                s3_client.upload_file(fullpath, bucket, filename)
+            except ClientError as e:
+                logging.error(e)
+                return False
```

Comment:
Perhaps we should add a 'finally' statement to delete the file from the container? I see what you mean now. It won't affect us in the short term, but I don't want to forget about it. We could also leave a TODO comment, similar to the one left in the generate_service_request.

Reply:
@lipsotiko Good idea. I've added local cleanup when the file is stored in S3.